sdk/python/packages/flet-core/src/flet_core/types.py

```diff
@@ -290,3 +297,20 @@ class MouseCursor(Enum):
     WAIT = "wait"
     ZOOM_IN = "zoomIn"
     ZOOM_OUT = "zoomOut"
+
+
+# Events
+ControlEventFunction = Callable[[ControlEvent], None]
```

Comment:
I think this should be `Optional`:

![image](https://github.com/flet-dev/flet/assets/5041459/61f7a051-a8fe-4a04-bd92-931d2349ab98)

Reply:
Ok, thanks. I'll fix that